### PR TITLE
Add pipx instructions to install guide

### PR DESCRIPTION
**Pull request by sivel**
##### SUMMARY
This PR aims to address some complexities of [PEP 668], by adding instructions for installing via `pipx`

[PEP 668]: https://peps.python.org/pep-0668/

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs/docsite/rst/installation_guide/intro_installation.rst

##### ADDITIONAL INFORMATION
As of now, `pipx` appears above `pip`, but can be easily moved.

### CODE REVIEW

File: docs/docsite/rst/installation_guide/intro_installation.rst
Comment (webknjaz) [inline docs/docsite/rst/installation_guide/intro_installation.rst:276]: @sivel is `ansible-community` a thing? O_0

**Review by webknjaz — APPROVED** [review]
:tada: :shipit: :rocket:

File: docs/docsite/rst/installation_guide/intro_installation.rst
Comment (sivel) [reply docs/docsite/rst/installation_guide/intro_installation.rst:276]: apparently.  I saw some note of it, and then decided to look into it

File: docs/docsite/rst/installation_guide/intro_installation.rst
Comment (samccann) [inline docs/docsite/rst/installation_guide/intro_installation.rst:121]: Why are the pipx instructions before pip?  Seems like most users will use pip so it should remain at the top, with pipx instructions below...

Comment (sivel) [reply docs/docsite/rst/installation_guide/intro_installation.rst:121]: > As of now, pipx appears above pip, but can be easily moved.

😄 

However, due to pep668, this could very well become the most prevalent means to install.  I suppose this is part of the discussion I need feedback on.

Comment (samccann) [reply docs/docsite/rst/installation_guide/intro_installation.rst:121]: ok if it's the 'wave of the future' I'm happy having it first... Let's see if anyone else comments before I merge.

**Comment by samccann** [comment]
Thanks @sivel  - is it okay if I add this to my next batch of 2.15 backports or is this a devel-only pr?

**Comment by sivel** [comment]
> Thanks @sivel - is it okay if I add this to my next batch of 2.15 backports or is this a devel-only pr?

Let's hold off on backporting for right now.

**Pull request merged**
